Makefile

```diff
@@ -89,7 +89,7 @@ endif
 	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
 
 
-.PHONY: release-src
-release-src:
+.PHONY: release
```

Comment:
Does release only do a package work? 

cc @moonming @ShiningRush

Reply:
Do you want to include the build process?